source/installation-guide.rst

```diff
@@ -9,8 +8,23 @@ Kubernetes and Google Cloud Platform support is on track to be implemented.
   :local:
 
 
-Installation - Docker compose
-=============================
+Hardware requirements
+---------------------
+
+In order to properly run dojot, the minimum hardware requirements are:
+
+- 4GB of RAM
```

Comment:
4 GB of RAM? Really? Only a minimalistic version

Reply:
iotmid03, gui-envs, etc have only 4GB of RAM....